### PR TITLE
[Redesign] Address PM pass design issues

### PR DESCRIPTION
**Pull request by joelverhagen**
Fixed:
- All pages: Transient messages e.g. unlisting a package should span the full 12 columns (or however many columns the page's contents take up)
  - This is fixed by telling the header Razor the page width in columns.
- Dev Environment: Warning Icon should be same height as the test for the banner
- Home Page: The 3 CTAs below the jumbotron should link to the pages
- Package Details:
   - Share and Tags Font size do not match the rest of the column
   - The show more for Version history is adding an extra bar when it is clicked
   - Copy button is not flush with commandline entry (off by about a pixel)
   - Spacing between owners name and avatar should be increased ( ~10px)
   - Icon is not centered to "Shore more/less"for expander for lists
   - Space Facebook and Twitter icons a bit more
   - Should there be an expander on the Dependencies section even if there are no dependencies? (yes)
   - Hover behavior over "Show more/less" is not consistent with other pages changes colors here, other pages, underlines (see Package Statistics). Should be consistent with other links (underline)
   - Pending edits notification has a button in the transient message, it should just be a link.
- Contact: This page should be a single column page spanning 10 columns (to make it work with the signed in version
- Account Settings:
  - Settings Entries (e.g. Email Address, Notifications, Password, etc.) should be spaced out a bit more currently it looks like ~10 px should be ~40px
  - Spacing between section header and the contents rectangle should be spaced ~5-10px more
  - More spacing between forms and buttons for each section ~20-40px
- API Keys: Buttons for each API key have font that is too big. Should match the size for Package controls (e.g. Manage my packages)

Missing:
- Home Page: Number callouts have a line between the rectangle and triangle

### CODE REVIEW

File: src/Bootstrap/dist/css/bootstrap-theme.css [hunk 1]
```diff
@@ -458,36 +464,26 @@ img.package-icon {
...
-.page-api-keys .api-key-details .icon-buttons a:hover {
-  text-decoration: none;
-}
 .page-package-details .no-border {
   border: 0;
 }
-.page-package-details .btn-expander:hover,
-.page-package-details .btn-expander:focus {
-  text-decoration: none;
+.page-package-details .btn-expander .ms-Icon {
```
Comment (ryuyu) [inline src/Bootstrap/dist/css/bootstrap-theme.css]: Can we move this to common? i think our other expander buttons share the problem that this fixes.

File: src/NuGetGallery/Views/Packages/DisplayPackage.cshtml [hunk 2]
```diff
@@ -263,8 +262,14 @@
                 }
                 else
                 {
-                    <h2>Dependencies</h2>
-                    <p>This package has no dependencies.</p>
+                    <h2>
+                        <a href="#" role="button" data-toggle="collapse" data-target="#dependency-groups"
```
Comment (ryuyu) [inline src/NuGetGallery/Views/Packages/DisplayPackage.cshtml]: Is there any difference between this Dependency collpaser and the one above other than the aria-expanded value? Can we make this above the if statement and set aria-expanded value to Model.Dependencies.DependencySets.Any()?

File: src/NuGetGallery/Views/Packages/DisplayPackage.cshtml [hunk 3]
```diff
@@ -357,11 +362,11 @@
                 </table>
                 @if (rowCount > 5)
                 {
-                    <button type="button" data-toggle="collapse" class="btn btn-link btn-expander" data-target="#hidden-versions"
+                    <a role="button" data-toggle="collapse" class="btn-expander" data-target="#hidden-versions"
```
Comment (ryuyu) [inline src/NuGetGallery/Views/Packages/DisplayPackage.cshtml]: We should add the icon-link class here that was added to base.

File: src/NuGetGallery/Views/Shared/Gallery/Header.cshtml [hunk 4]
```diff
@@ -128,7 +145,20 @@
     <div class="alert-transient @(ViewBag.HasJumbotron == true ? "alert-transient-jumbotron" : string.Empty)">
         <div class="container">
             <div class="row">
-                <div class="col-sm-offset-2 col-sm-8">
+                @{
```
Comment (ryuyu) [inline src/NuGetGallery/Views/Shared/Gallery/Header.cshtml]: With this change, should all pages be defining the corresponding ViewBag properties?

File: src/NuGetGallery/Views/Packages/DisplayPackage.cshtml (hunk 2 above)
Comment (joelverhagen) [inline src/NuGetGallery/Views/Packages/DisplayPackage.cshtml]: Good idea.

File: src/NuGetGallery/Views/Packages/DisplayPackage.cshtml (hunk 3 above)
Comment (joelverhagen) [inline src/NuGetGallery/Views/Packages/DisplayPackage.cshtml]: Yep, took that from the merge.

File: src/NuGetGallery/Views/Shared/Gallery/Header.cshtml (hunk 4 above)
Comment (joelverhagen) [inline src/NuGetGallery/Views/Shared/Gallery/Header.cshtml]: Yeah.

File: src/NuGetGallery/Views/Authentication/LinkExternal.cshtml [hunk 5]
```diff
@@ -2,6 +2,8 @@
 @{
     ViewBag.Title = "Sign In";
     ViewBag.Tab = "Sign in";
+    ViewBag.SmPageColumns = 6;
```
Comment (joelverhagen) [inline src/NuGetGallery/Views/Authentication/LinkExternal.cshtml]: Fixed.

**Pull request merged**